static/carousel/gruntfile.js

```diff
@@ -38,11 +41,11 @@ module.exports = function(grunt) {
         }
       },
       uglify: {
-        options: {
-          compress: {
-            drop_console: true
-          }
-        },
```

Comment:
Please revert this change, we will check if this does not work for everyone first and if necessary we will need to find a different package to do this task